### DISCUSSION TITLE
I have recently faced rendering issues with my Github account

### DISCUSSION THREAD

**Discussion by pravin1101** [community/community#36425] [Other Feature Feedback, Questions, & Ideas] [answered]
I coudn't view any pdf files , image files and if I download it it shows this error

![Screenshot (31)](https://user-images.githubusercontent.com/66785258/196194228-7af389b9-6146-458c-baba-6c24ee0312a8.png)
![Screenshot (32)](https://user-images.githubusercontent.com/66785258/196194345-4642ebc7-d072-4c22-965f-0b724e9d775a.png)

How can i Solve it and can anyone give me a solution for this

**Comment by jfullstackdev** [comment]
that loaded to me quite a bit but never slow

![image](https://user-images.githubusercontent.com/47092464/196222169-7529e36a-7973-468d-811e-ecc34078dcca.png)

maybe your computer memory, firewall / antivirus slowing it down, slow network connection 

or some extensions installed affecting it, try it in Incognito mode

**Reply by pravin1101** [reply]
Thanks, For the help I tried incognito mode but it didn't solve the problem... 
any other suggestions

**Comment by airtower-luna** [comment, marked as answer]
It looks like you have a DNS problem, and your browser can't find the server where GitHub hosts that PDF. You'll have to investigate your DNS servers (usually your router, or at your internet provider) to see what's wrong, or ask an admin if you're on someone else's network (e.g. at a company or university).

As a quick fix you could try to configure an alternative DNS server on your computer, e.g. [Quad9](https://quad9.net/).

**Reply by pravin1101** [reply]
Yes after switching my network I got my pdf rendered

**Reply by pravin1101** [reply]
But I am still facing issues in downloading the files

**Comment by ossanna16** [comment]
Hey there @pravin1101 and thank you for asking a good question! I'm glad you found our community 🙂

**Reply by pravin1101** [reply]
I am glad to be a part of the community
